### PR TITLE
Angi api

### PR DESCRIPTION
**Pull request by ilonalink**

### CODE REVIEW

File: cypress/tests/apiAndIntegrations/angi.spec.ts [hunk 1]
```diff
@@ -20,12 +21,14 @@ describe('Custom documents test',() => {
         let leadId = RandomFunctions.generateRandomPhone();
         let leadName = RandomFunctions.generateRandomGroupName();
 
-        angiPage.sendAngiCreateRequest(leadId,leadName);
+        ApiRequests.createAngiApi(leadId,leadName);
```
Comment (IdoAdler2) [inline cypress/tests/apiAndIntegrations/angi.spec.ts:24]: @ilonalink It should be generic class of api request.
Change name to apirequests. not to angi requests.

File: cypress/tests/apiAndIntegrations/angi.spec.ts [hunk 2]
```diff
@@ -20,12 +21,14 @@ describe('Custom documents test',() => {
...
         cy.get('@res').then((response) => {
```
Comment (IdoAdler2) [inline cypress/tests/apiAndIntegrations/angi.spec.ts:25]: @ilonalink give the @res specific name @angiResponse

Comment (ilonalink) [reply cypress/tests/apiAndIntegrations/angi.spec.ts:25]: @IdoAdler2 fixed

File: cypress/support/apiRequests.ts [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+                "stateProvince": "GA",
+                "postalCode": "12345",
+                "primaryPhone": "5555555559",
+                "phoneExt":"1234",
+                "secondaryPhone": "5555555558",
+                "secondaryPhoneExt":"1234",
+                "email": "eran.peled@workiz.com",
+                "srOid": 876543112221,
+                "leadOid": leadId,
+                "taskOid": 40006,
```
Comment (IdoAdler2) [inline cypress/support/apiRequests.ts:25]: @ilonalink move all the body to another place.

inside infra > open folder called, apiBodayTemplats

File: cypress/support/apiRequests.ts [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+                    "appointmentOid": 8668,
+                    "status": "Confirmed",
+                    "type": "Service",
+                    "start": "2022-06-09T12:00:00.000-07:00",
+                    "end": "2022-06-09T14:00:00.000-07:00",
+                    "external_id": "HA-b97290e2-1441-466a-abb3-e2fda9a94cbc.ics"
+                }
+                }
+        }).then((res) => {
+            console.log(res.body["flag"]);
```
Comment (IdoAdler2) [inline cypress/support/apiRequests.ts:63]: @ilonalink remove console. log

**Review by IdoAdler2 — APPROVED** [review]

**Pull request merged**